Rubberduck.sln

```diff
@@ -643,6 +649,36 @@ Global
 		{AC54B7FB-170D-4DA6-A30B-8CAD182F0E6B}.Release64|x64.Build.0 = Release|Any CPU
 		{AC54B7FB-170D-4DA6-A30B-8CAD182F0E6B}.Release64|x86.ActiveCfg = Release|Any CPU
 		{AC54B7FB-170D-4DA6-A30B-8CAD182F0E6B}.Release64|x86.Build.0 = Release|Any CPU
+		{68F55404-32CA-4414-96B8-E284F3B1F846}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{68F55404-32CA-4414-96B8-E284F3B1F846}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{68F55404-32CA-4414-96B8-E284F3B1F846}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{68F55404-32CA-4414-96B8-E284F3B1F846}.Debug|x64.Build.0 = Debug|Any CPU
+		{68F55404-32CA-4414-96B8-E284F3B1F846}.Debug|x86.ActiveCfg = Debug|Any CPU
+		{68F55404-32CA-4414-96B8-E284F3B1F846}.Debug|x86.Build.0 = Debug|Any CPU
+		{68F55404-32CA-4414-96B8-E284F3B1F846}.Debug64|Any CPU.ActiveCfg = Debug|Any CPU
+		{68F55404-32CA-4414-96B8-E284F3B1F846}.Debug64|Any CPU.Build.0 = Debug|Any CPU
+		{68F55404-32CA-4414-96B8-E284F3B1F846}.Debug64|x64.ActiveCfg = Debug|Any CPU
+		{68F55404-32CA-4414-96B8-E284F3B1F846}.Debug64|x64.Build.0 = Debug|Any CPU
+		{68F55404-32CA-4414-96B8-E284F3B1F846}.Debug64|x86.ActiveCfg = Debug|Any CPU
+		{68F55404-32CA-4414-96B8-E284F3B1F846}.Debug64|x86.Build.0 = Debug|Any CPU
+		{68F55404-32CA-4414-96B8-E284F3B1F846}.DebugAccess|Any CPU.ActiveCfg = Debug|Any CPU
+		{68F55404-32CA-4414-96B8-E284F3B1F846}.DebugAccess|Any CPU.Build.0 = Debug|Any CPU
+		{68F55404-32CA-4414-96B8-E284F3B1F846}.DebugAccess|x64.ActiveCfg = Debug|Any CPU
+		{68F55404-32CA-4414-96B8-E284F3B1F846}.DebugAccess|x64.Build.0 = Debug|Any CPU
+		{68F55404-32CA-4414-96B8-E284F3B1F846}.DebugAccess|x86.ActiveCfg = Debug|Any CPU
+		{68F55404-32CA-4414-96B8-E284F3B1F846}.DebugAccess|x86.Build.0 = Debug|Any CPU
+		{68F55404-32CA-4414-96B8-E284F3B1F846}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{68F55404-32CA-4414-96B8-E284F3B1F846}.Release|Any CPU.Build.0 = Release|Any CPU
+		{68F55404-32CA-4414-96B8-E284F3B1F846}.Release|x64.ActiveCfg = Release|Any CPU
+		{68F55404-32CA-4414-96B8-E284F3B1F846}.Release|x64.Build.0 = Release|Any CPU
+		{68F55404-32CA-4414-96B8-E284F3B1F846}.Release|x86.ActiveCfg = Release|Any CPU
+		{68F55404-32CA-4414-96B8-E284F3B1F846}.Release|x86.Build.0 = Release|Any CPU
+		{68F55404-32CA-4414-96B8-E284F3B1F846}.Release64|Any CPU.ActiveCfg = Release|Any CPU
+		{68F55404-32CA-4414-96B8-E284F3B1F846}.Release64|Any CPU.Build.0 = Release|Any CPU
+		{68F55404-32CA-4414-96B8-E284F3B1F846}.Release64|x64.ActiveCfg = Release|Any CPU
+		{68F55404-32CA-4414-96B8-E284F3B1F846}.Release64|x64.Build.0 = Release|Any CPU
+		{68F55404-32CA-4414-96B8-E284F3B1F846}.Release64|x86.ActiveCfg = Release|Any CPU
+		{68F55404-32CA-4414-96B8-E284F3B1F846}.Release64|x86.Build.0 = Release|Any CPU
 	EndGlobalSection
```

Comment:
These configs reek of being accidentally carried over from an ancient version of the project; don't we need nothing but `Debug` and `Release` (any CPU)?

Reply:
I honestly don't know. But that particular section has a few sets of this configuration setup... I'm not sure I want to deal with that in this PR